code/platformio_ota.ini

```diff
@@ -39,6 +39,9 @@ extends = env:itead-sonoff-pow
 [env:itead-sonoff-pow-r2-ota]
 extends = env:itead-sonoff-pow-r2
 
+[env:itead-sonoff-pow-r3-ota]
+extends = env:itead-sonoff-pow-r33
```

Comment:
Typo: `env:itead-sonoff-pow-r33` should be `env:itead-sonoff-pow-r3`

Reply:
Thanks for spotting the typo - has been fixed.